### PR TITLE
Add TOS acceptance field to /me/

### PR DESCRIPTION
**Pull request by RuthShryock**
## Checklist

1. [X] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [X] Ensure the tests pass
4. [X] Make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [X] Write a title and, if necessary, a description of your work suitable for publishing in our [release notes](https://community.kobotoolbox.org/tag/release-notes)
6. [X] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes

## Description

API edit for TOS feature

## Notes

Added `accepted_tos` boolean which is populated based on whether the user has made a POST request to the`/me/tos` endpoint

### CODE REVIEW

File: kpi/serializers/current_user.py
Comment (bufke) [inline kpi/serializers/current_user.py:107]: Add a docstring here just explaining what this does. It may not be obvious that this is a simplified and private property on the API. Spell out terms of service in these docs too.

**Review by bufke — APPROVED** [review]
Approved pending my documentation request - good to merge otherwise.

**Comment by RuthShryock** [comment]
This is failing because the test makes a call to the [ToS api](https://github.com/kobotoolbox/kpi/pull/4712) which is not yet merged into the feature/tos branch. It should pass after the merge and updates to this branch.

**Pull request merged**
